src/centralized_telescope/mod.rs

```diff
@@ -0,0 +1,20 @@
+//! ALBA's bounded DFS scheme using Blake2b as hash function.
+//! (c.f. Section 3.2.2 of Alba paper)
+
+// Prove and Verify functions
+pub mod algorithm;
+
+// Setup function
+pub mod init;
+
+// Structures
```

Comment:
nit: it wasn't super clear that this applies to the four items below, maybe remove empty lines in between?